core/src/main/java/google/registry/flows/domain/DomainFlowUtils.java

```diff
@@ -1501,6 +1512,13 @@ public NotAuthorizedForTldException(String tld) {
     }
   }
 
+  /** Registrar is missing the billing account map for this currency type. */
+  public static class MissingBillingAccountMapException extends AuthorizationErrorException {
+    public MissingBillingAccountMapException(CurrencyUnit currency) {
+      super("Registrar is not yet onboarded for TLDs that bill in " + currency);
```

Comment:
Can we make this 'Registrar is not **fully** onboarded..' ? I'd like to cover edge cases in which something changes following an originally successful onboarding